### DISCUSSION TITLE
Deadbeef does not read cd metadata, but Clementine does.

### DISCUSSION THREAD

**Discussion by circuitikzuser** [DeaDBeeF-Player/deadbeef#2744] [Q&A]
For some reason Deadbeef does not read and find the title/creator and track names for cdemu images, which Clementine does find for the exact same image.
Clementine has other issues, which makes deadbeef preferable and I dont want to continue with clementine.

How do I resolve the issue that clearly all the data and metadata is clearly available, as Clementine shows correctly and reads, but deadbeef doesnt using exactly the same image. "

Here is the image showing the issue.
[metaissuesdeadbeefvsclementine](https://jumpshare.com/v/VZJJ2pQKDjP3k2Rpik9j)

**Comment by Oleksiy-Yakovenko** [comment]
One possible reason would be a move from freedb.org to gnudb.org
deadbeef is using freedb.org URL by default (you can find it in the Audio CD Player plugin settings).
It looks like it automatically redirects to gnudb.org, but maybe the cddb library can't handle redirects.
Could you try changing to gnudb.org and see whether it changes anything?
I guess, you would need to re-add the cd image to playlist to force-reload the new metadata

**Reply by Oleksiy-Yakovenko** [reply]
Actually, it seems the correct URL to use is `gnudb.gnudb.org` according to this page:
https://gnudb.org/howto.php

**Reply by Oleksiy-Yakovenko** [reply]
You might also need to change the port to 80

**Reply by circuitikzuser** [reply]
Everything was as required except the port was 888.
I changed it to 80, but it made absolutely no difference.
I ran reload metadata and nothing changed.
I closed it and reopened deadbeef and nothing changed.
I also tried "prefer CD text" in the Audio CD Plugin  and it did not help.

I would have thought the problem is with my cd image and TOC, but as mentioned Clementine has no problem with the exact TOC and img.
Would it help if I post the TOC ?
I can post the image too but definitely not publicly and have to give you the link through pm if you want to use that to check.
I noticed that k3b e.g. has what seems like binary TOC, while programs like Brasero create plain text TOC. Clementine does not have a problem with either plain text or binary.
I noticed that images that deadbeef used to recognized the TOCs for now doesnt. I was fooled by the fact that the metadata was seemingly saved somewhere with the interface config when it used to be able to read it and then use that rather than reading the CD, thereafter, almost like cache. 
If I create a new playlist  and  uses exactly the same tracks in the new playlist then it does not show track titles and names, while my old playlists in deadbeef using the identical image and TOC shows them.
I should add that the TOC and img as used in the original post was totally new to both deadbeef and clementine and there could not be any pre-existing cache.
So this must have happened with the new deadbeef I installed recently namely 1.8.8. or I am doing something wrong, but I cannot see what. Maybe there are missing dependencies or plugins ?
I might have to go to back to the older version.

**Reply by Oleksiy-Yakovenko** [reply]
Have you tried removing the tracks from playlist and adding the CD image again?

**Comment by circuitikzuser** [comment]
I tried gnudb.gnudb.org and port 80, still doesnt work.
It seems gnudb requires email address and password from the page you referenced.
Maybe they require that recently.
There are no such username password fields in the Audio CD Player  plugin I can configure if that is what gnudb require.

**Comment by circuitikzuser** [comment]
I sent you a video to your "waker" email address showing what I encounter. 
The old playlist seemingly had cash and has the track and album name displayed
Opening the same CD in 1.8.8. in a new playlist shows no track names etc and reverts to device name clearly not reading the cd toc /metadata or doesnt quiery the online database.
The two playlists I show in the video uses the identical same image and toc.

**Reply by Oleksiy-Yakovenko** [reply]
Well, if you can confirm that metadata for this CD image loads correctly in older version of deadbeef -- such information would be useful to help tracking down the problem.

**Comment by circuitikzuser** [comment]
I will do that. I still have the home directory and Linux partition for the older system archived so I will extract them to partitions and boot it up and report back. Might be some time.

In the mean time, Is there anything wrong from my side that you could think that could cause this, I could look at and fix.?

**Reply by Oleksiy-Yakovenko** [reply]
It's cool that you're gonna have fun with partitions at all, but for the purpose of trying an older version of deadbeef - you could have just downloaded a build and ran it :)
do you remember which version number has worked for you in the past?
All 1.8.x builds are available here:
https://sourceforge.net/projects/deadbeef/files/travis/linux/

And if you want to go even older they're here:
https://sourceforge.net/projects/deadbeef/files/Archive/

**Reply by Oleksiy-Yakovenko** [reply]
> In the mean time, Is there anything wrong from my side that you could think that could cause this, I could look at and fix.?

Currently, there's no way to tell for me whether you can do anything else.

**Reply by circuitikzuser** [reply]
It is no problem getting the entire old system up and running and deadbeef worked great on the same cd images so i want to use a reference that I know worked.
I can only see the version when I boot up the old system. So that is the only way.
If for any reason the old system also gives me trouble now, then we know it is something to do with recent changes at gnudb, so it is a good idea to check the old system, which is known to work.
Also if there is a difference between how the same old version reads metadata on the old system and new system using the same version of deadbeef we know it is something to do with the new linux distro & deadbeef.
Once I compared the old system with that of 1.8.8 I will try all the older versions.
Thanks for all the help.
I will boot the old system in the next few days.

**Comment by circuitikzuser** [comment]
Update:
It will still be some time before I can boot as the server is in  use. I will update this thread as soon as I get time to boot.
